lua/examples/example_gt_lpm_params_request.lua

```diff
@@ -0,0 +1,23 @@
+require "gatekeeper/staticlib"
+require "gatekeeper/lpmlib"
+
+local reply_msg = ""
+
+local dyc = staticlib.c.get_dy_conf()
+
+if dyc.gt == nil then
+	return "Gatekeeper: failed to run as Grantor server\n"
+end
+
+-- The function assumes that the variable lpm, an IPv4 LPM table,
+-- is globally available in the policy as in the policy example.
+local function example()
+	local max_rules, number_tbl8s = lpmlib.lpm_get_paras(lpm)
+	return lpmlib.c.gt_lcore_id() .. ":" .. max_rules ..
+		"," .. number_tbl8s .. "\n"
+end
+
+reply_msg = reply_msg .. dylib.update_gt_lua_states_incrementally(
+	dyc.gt, example, true)
```

Comment:
Add the qualifier `local` to the beginning of this assignment, drop `reply_msg .. `, and drop the lonely line `local reply_msg = ""`at the beginning of the file.